docs/installation/advanced_topics.asciidoc

```diff
@@ -30,40 +30,67 @@ In order to override a default parameter, define it and set it in [filename]`/us
+* `description`*: A user-friendly description that provides more details about the report. Used as a title for all charts.
+* `charts`: A comma delimited list of charts to display. Each chart is displayed in its own tab above the table data. There is no limit to the number of charts that can be defined. Charts are explained in more detail below.
+* `columns`*: A comma separated list of columns or aliases that are displayed in the table from the SQL query (ex: `node.mac`, `Node MAC`). The table columns are displayed in the respective order. Columns can be aliased to a more friendly name, but these aliases must be used throughout the other attributes.
+* `formatting`: A comma separated list of column or alias formatters. Each column is defined folowed by a colon and the internal PacketFence function used to format the column value for every row (ex: `formatting=vendor:oui_to_vendor`). This is used to format the query result columns using a function to access internal PacketFence memory. The supported formatters:
+  ** `oui_to_vendor`: format a MAC OUI to a vendor.
+* `has_date_range`: _[enabled|disabled]_ Display a datetime range and provide `start_date` and `end_date` bindings.
+* `has_limit`: _[enabled|disabled]_  Display a limit selection and provide a `limit` binding.
+* `node_fields`: A comma delimited list of fields (columns or aliases) that will be clickable from the table of the Report and linked to the specific Node - only clickable if the reports' user has the _"Node - View"_ admin role. All fields must be a valid PacketFence node identifier _(mac)_.
+* `person_fields`: A comma delimited list of fields (columns or aliases) that will be clickable from the table of the Report and linked to the specific User - only clickable if the reports' user has the _"User - View"_ admin role. All fields must be a valid PacketFence user identifier _(pid)_`.
+* `role_fields`: A comma delimited list of fields (columns or aliases) that will be clickable from the table of the Report and linked to the specific Role. All fields must be a valid PacketFence role identifier _(category_id)_.
+
+NOTE: Configuration attributes can optionally use a `columnName` reference with simple queries that use a single table (ex: `attribute=columnA,columnB`). The attributes must use `tableName.columnName` reference when using `joins` with 2+ tables (ex: `attribute=tableA.columnA,tableB.columnB`). Aliased `columns` can be used with the table reference (ex: `attribute=tableA.Alias A,tableB.Alias B`).
+
+==== SQL Abstract
+
+When `type=abstract` PacketFence uses Perl https://metacpan.org/pod/SQL::Abstract::More[SQL::Abstract::More] to automatically build the SQL query.
+
+The following attributes are available when using `type=abstract`(* mandatory attributes are marked with an asterisk): [options="compact"]
```

Comment:
`[options="compact"]` is an asciidoc list setting; move it on its own line, right before the list definition.